### ISSUE TITLE
Unify workshop content between ExaCS and ExaCC

### ISSUE THREAD

**Issue by albertyckwok** [tejuscs/exacs-QA#4]
This is an start of discussion of unify some content to make the same labs work on both ExaCS and ExaCC whenever possible. I have done it with DBVault and will create a branch for that. Any comment/suggests/QA helps if highly welcome and appreciated.
Thanks very much in advance
Albert

**Comment by tejuscs** [comment]
Yes, it absolutely makes sense to unify. We can discuss this sometime tomorrow over a call. 

Thanks,
Tejus
